### DISCUSSION TITLE
Setting subscription of Apple Store and HUAWEI.

### DISCUSSION THREAD

**Discussion by kanisuki** [airsdk/Adobe-Runtime-Support#3074] [Q&A]
Hello,
I have almost done to set subscription to Google Play.
I wrote several lines of code and added 1 page with several lines of code for subscription in fla and app.xml and set distriqt ANE.
And after this I input fields in subscription pages in Google Play Console.
About Apple Store and HUAWEI,
Are there any differences in case of Google Play?
Are there any webpages or YouTube?

**Comment by marchbold** [comment]
If you require support for our extensions you can ask our developers directly in the extensions repository: 

https://github.com/distriqt/ANE-InAppBilling/issues/new

**Reply by kanisuki** [reply]
Thank you.
